--- brlaser.drv.in.orig	2021-01-29 20:22:32.833757102 +0100
+++ brlaser.drv.in	2021-01-13 23:06:04.383382397 +0100
@@ -143,6 +143,14 @@
 }
 
 {
+  ModelName "DCP-7070DW"
+  Attribute "NickName" "" "Brother DCP-7070DW, $USING"
+  Attribute "1284DeviceID" "" "MFG:Brother;CMD:PJL,PCL,PCLXL;MDL:DCP-7070DW;CLS:PRINTER;CID:Brother Laser Type1;"
+  Duplex rotated
+  PCFileName "br7070dw.ppd"
+}
+
+{
   ModelName "DCP-7080"
   Attribute "NickName" "" "Brother DCP-7080, $USING"
   Attribute "1284DeviceID" "" "MFG:Brother;CMD:PJL,HBP;MDL:DCP-7080;CLS:PRINTER;CID:Brother Laser Type1;"
